### ISSUE TITLE
School project

### ISSUE THREAD

**Issue by merijnverlee** [A-Wheeto/Dashboard#5]
Hey, i am using your project as an example for my school project which is about Steam. I just can't figure out how to import all the images from the internet and stuff. Can you maybe help me out so i can finish my school project? I only have 3 weeks left and been working on it for 4 weeks.

**Comment by A-Wheeto** [comment]
Hello, I made my dashboard project 4 years ago using Beautiful Soup to web scrape the contents from the Steam homepage.
It's possible the HTML and CSS classes on the Steam page have changed by now and broken my code - I haven't tried it yet.
I believe I'm using PIL (Python Image https://pillow.readthedocs.io/en/stable/handbook/tutorial.html) to show the images from the website after retrieving the URLs.
Are you using Tkinter to draw the images the same way I did?

**Comment by merijnverlee** [comment]
Yes, i am sort of just trying to replicate the steam top trending box and the news box but it almost looks impossible to make it the same way you did.
________________________________
Van: Adam Wheatley ***@***.***>
Verzonden: maandag 23 december 2024 12:26
Aan: A-Wheeto/Dashboard ***@***.***>
CC: Merijn Verlee ***@***.***>; Author ***@***.***>
Onderwerp: Re: [A-Wheeto/Dashboard] School project (Issue #5)


Hello, I made my dashboard project 4 years ago using Beautiful Soup to web scrape the contents from the Steam homepage.
It's possible the HTML and CSS classes on the Steam page have changed by now and broken my code - I haven't tried it yet.
I believe I'm using PIL (Python Image https://pillow.readthedocs.io/en/stable/handbook/tutorial.html) to show the images from the website after retrieving the URLs.
Are you using Tkinter to draw the images the same way I did?

**Comment by A-Wheeto** [comment]
Can you share with me what you've got so far and I'll see if I can help. Thank you.
